__tests__/external/ACSAdapter/AMSFileManager.spec.ts

```diff
@@ -99,7 +108,7 @@ describe('AMSFileManager', () => {
         expect(response).toStrictEqual(amsMetadata);
     });
 
-    it('AMSFileManager.getFileMetadata() should return a nothing if invalid', async () => {
+    it('AMSFileManager.getFileMetadata() should return nothing if invalid', async () => {

```

Comment:
same question  about error code with invalid scenarios

Reply:
It's a valid scenario as explained previously